src/org/zaproxy/zap/extension/portscan/resources/Messages.properties

```diff
@@ -2,7 +2,7 @@
 
 ports.copy.popup                = Copy
 ports.desc                      = Simple but effective port scanner
-ports.options.label.maxPort     = Maximum port to scan:
+ports.options.label.maxPort     = Maximum port(s) to scan:
```

Comment:
Ok you're right, I made a poor assumption based on other port scanners. How about we change it to `Highest port number to scan:` then?